examples/webgl_animation_skinning_ik.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - animation - skinning - ik</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<meta name="author" content="Antoine BERNIER (abernier)" />
+		<link type="text/css" rel="stylesheet" href="main.css">
+		<style>
+		body {color:white;}
+		#info a {
+			color:inherit;
+		}
+		</style>
+	</head>
+	<body>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - <a href="https://assetstore.unity.com/packages/3d/characters/humanoids/humans/kira-lowpoly-character-100303" target="_blank" rel="noopener">Kira</a>'s "<a href="https://en.wikipedia.org/wiki/Hand_with_Reflecting_Sphere" target="_blank" rel="noopener">hand with Reflecting Sphere</a>"<br />
+			(see <a href="https://abernier.name/three.js/examples/webgl_kira.html" target="_blank" rel="noopener">experiment</a>)
```

Comment:
It seems the link to `https://abernier.name/three.js/examples/webgl_kira.html` is broken.

Reply:
lemme fix that